reana_commons/config.py

```diff
@@ -446,3 +446,8 @@ def default_workspace():
 
 REANA_DEFAULT_SNAKEMAKE_ENV_IMAGE = "snakemake/snakemake:v6.8.0"
 """Snakemake default job environment image."""
+
+REANA_JOB_CONTROLLER_CONNECTION_CHECK_SLEEP = int(
+    os.getenv("REANA_JOB_CONTROLLER_CONNECTION_CHECK_SLEEP", "10")
```

Comment:
Question: Shall we allow float values as [here](https://github.com/reanahub/reana-server/blob/4c986b2847f546ee3eaac3a1d752dd77a2ff4ed6/reana_server/config.py#L183)?